.env.example

```
@@ -0,0 +1,11 @@
+CY_WRONG_EMAIL="xxx@gmail.com"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)